style.css

```diff
@@ -219,56 +238,63 @@ img {vertical-align: middle;}
 
 .about_us_A  {
 	margin-top: 50px;
-	padding: 35px 0px 35px 50px;
-	background-color: #BEB8D2;
+	background-color: white;
+	width: 100%;
 	grid-column-start: 2;
 	grid-column-end: 5;
 	text-align: center;
 	z-index: 30;
 	grid-row: 3;
-	height: auto;
+    float: left;
+	max-height: 300px;
+	max-width: 300px;
+	min-height: 300px;
+	min-width: 300px;	
 }
 
 .about_us_A img {
-	padding: ;
-	object-fit: cover;
-	height: auto;
-	margin: 0px 0px 0px 10%;
+    height: 100%;
 	width: 100%;
-	height: auto;
-	display: inline-block;
+	object-fit: cover;
 	border-radius: 200px;
+    display: block;
 }
 
 /* Textual section in about_Us (index.html) */
 
 .about_us_B {
 	margin-top: 50px;
 	padding: 20px 0px 0px 0px;
-	height: auto;
-	grid-column-start: 5;
+	max-height: 350px;
+	min-height: 300px;
+	grid-column-start: 2;
 	grid-column-end: 14;
 	grid-row: 3;
-	background-color: #BEB8D2;
+	background-color: white;
 	text-align: left;
+	z-index: 29;
+	box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 8px 6px 20px 10px rgba(0, 0, 0, 0.19);
 }
 
 .about_us_B h1{

```

Comment:
This font is ok, i might decide to increase the font-weight, however may need to a better one for this, however it is clear